src/OSPSuite.Core/Converter/v9/Converter730To90.cs

```diff
@@ -0,0 +1,90 @@
+using OSPSuite.Core.Serialization;
+
+namespace OSPSuite.Core.Converter.v9
+{
+   public class Converter730To90 : IObjectConverter
+   {
+      private readonly IDimensionFactory _dimensionFactory;
+
+      public Converter730To90(IDimensionFactory dimensionFactory)
+      {
+         _dimensionFactory = dimensionFactory;
+      }
+
+      public bool IsSatisfiedBy(int version) => version == PKMLVersion.V7_3_0;
+
+      public (int convertedToVersion, bool conversionHappened) Convert(object objectToUpdate)
+      {
+         return (PKMLVersion.V9_0, false);
+      }
+
+      public (int convertedToVersion, bool conversionHappened) ConvertXml(XElement element)
+      {
+         var conversionHappened = false;
+         convertDimensionIn(element, ref conversionHappened);
+         return (PKMLVersion.V9_0, conversionHappened);
+      }
+
+      private void convertDimensionIn(XElement element, ref bool conversionHappened)
+      {
+         //retrieve all elements with an attribute dimension
+         var allDimensionAttributes = from child in element.DescendantsAndSelf()
+            where child.HasAttributes
+            let attr = child.Attribute(Constants.Serialization.Attribute.Dimension)
+            where attr != null
+            select attr;
+
+
+         var allMappedDimensionIds = new HashSet<string>();
+         foreach (var attribute in allDimensionAttributes)
+         {
+            string attributeValue = attribute.Value;
+            if (int.TryParse(attribute.Value, out _))
+               allMappedDimensionIds.Add(attributeValue);
+            else
+            {
+               var newDimensionName = mapDimensionName(attributeValue);
+               if (!string.Equals(newDimensionName, attributeValue))
+               {
+                  attribute.SetValue(newDimensionName);
+                  conversionHappened = true;
+               }
+            }
+         }
+
+
+         var allMapAttributes = from child in element.Descendants(Constants.Serialization.STRING_MAP)
+            let id = child.Attribute(Constants.Serialization.Attribute.ID)
+            where allMappedDimensionIds.Contains(id.Value)
+            select child.Attribute(Constants.Serialization.Attribute.STRING);
+
+         foreach (var attribute in allMapAttributes)
+         {
+            var oldDimensionName = attribute.Value;
+            var newDimensionName = mapDimensionName(attribute.Value);
+            if (!string.Equals(oldDimensionName, newDimensionName))
+            {
+               attribute.SetValue(newDimensionName);
+               conversionHappened = true;
+            }
+         }
+      }
+
+      private string mapDimensionName(string dimensionName)
+      {
+         if(_dimensionFactory.Has(dimensionName))
+            return dimensionName;
+
+         if (dimensionName.IsOneOf($"{Constants.Dimension.DIMENSIONLESS} per Time", $"{Constants.Dimension.DIMENSIONLESS} per time", "Fraction per time", "Fraction per Time"))
```

Comment:
@pavelbal @Yuri05 
Here, I am trying convert some of the stupid dimensions to the existing one. 
Do we have anything else? in short, `dimensionless per time` and `fraction per time` will be converted to `Inversed time`